cmd/uber-apiserver/app/options/options.go

```diff
@@ -0,0 +1,213 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package options contains flags and options for initializing an apiserver
+package options
+
```

Comment:
It seems he's following @lavalamp's suggestion [here](https://github.com/kubernetes/kubernetes/pull/20733#issuecomment-181636776). I'm not sure what's the reasoning behind it. Could @lavalamp or @jianhuiz provide more context?